src/test.py

```diff
@@ -0,0 +1 @@
+
```

Comment:
if you plan to have unittest, which I can only support, maybe put them in tests/ at the same depths as scripts/ and src/. This way you can have the test, but you could also have a simple way not to ship them in a production setup

Reply:
that was something I created to get the hang of argparse, will delete